src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1223,7 +1223,7 @@ null, null, new MItemBlock(modElement.getWorkspace(),
 			PotionEffect potionEffect = new PotionEffect(modElement);
 			potionEffect.effectName = modElement.getName() + " Effect Name";
 			potionEffect.color = Color.magenta;
-			potionEffect.icon = "test.png";
+			potionEffect.icon = "test";

```

Comment:
Keep this at test.png if we agreed no backend changes

Reply:
Also not sure how you could change things so fast and also test them well. Remember, programming is about quality, not quanitity

Reply:
Oops, forgot once again tests 😅

Reply:
Also another thing, check "Files changed" tab on PR as a "sanity check" to make sure all seems in place as a sort of last check. You can also do this in IDEA in push dialog right before pushing